src/components/issues/Card.tsx

```diff
@@ -126,26 +231,77 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                 </div>
             </div>
             <div className={styles.actions}>
-                {(!taskExists || !isUserAuthorized || !devMode) && (
-                    <button
-                        className={styles.card__top__button}
-                        disabled={taskExists || isLoading || !isUserAuthorized}
-                        onClick={handleClick}
-                    >
-                        Convert to task
-                    </button>
-                )}
-                {isUserAuthorized && taskExists && devMode && (
-                    <ActionForm
-                        taskId={taskId || ''}
-                        createTask={async () => {
-                            console.log('Placeholder');
-                        }}
-                        updateTask={async () => {
-                            console.log('Placeholder');
-                        }}
-                        taskAssignee=""
-                    />
+                {devMode ? (
+                    <>
+                        <button
+                            className={styles.card__top__button}
+                            disabled={
+                                isLoading ||
+                                (!isUserAuthorized &&
+                                    (taskExists || !!requestId))
+                            }
+                            onClick={toggle}
+                        >
+                            {isUserAuthorized
+                                ? 'Convert to Task'
+                                : 'Request as Task'}
+                        </button>
```

Comment:
- please abstract this out into a separate component this will make it easier to make changes to it in the future.

Reply:
+1

Reply:
Done.